src/modules/accommodations/entities/accommodation.entity.ts

```diff
@@ -0,0 +1,71 @@
+import {
+  AccommodationCategory,
+  AccommodationStatus,
+} from 'src/shared/constants';
+import {
+  Column,
+  CreateDateColumn,
+  Entity,
+  Index,
+  OneToMany,
+  PrimaryGeneratedColumn,
+  UpdateDateColumn,
+} from 'typeorm';
+import { Amenity } from './amenity.entity';
+
+@Entity()
+export class Accommodation {
+  @PrimaryGeneratedColumn('uuid')
+  id: string;
+
+  @Column({
+    type: 'varchar',
+    length: 125,
+  })
+  name: string;
+
+  @Column({
+    type: 'varchar',
+    length: 255,
+  })
+  description: string;
+
+  @Column({
+    type: 'varchar',
+    length: 200,
+  })
+  location: string;
+
+  @Column({
+    name: 'price_per_night',
+    type: 'decimal',
+  })
+  price_per_night: number;
+
+  @Column({
```

Comment:
We had decided to create new table for category ?

Reply:
No, last time you said no need to do this

Reply:
After Davranbek raised question about handling it in frontend, I said let's have separate table